servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/ExecutionContextUtils.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright © 2022 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.servicetalk.http.netty;
+
+import io.servicetalk.http.api.DelegatingHttpExecutionContext;
+import io.servicetalk.http.api.HttpExecutionContext;
+import io.servicetalk.transport.api.IoExecutor;
+
+import io.netty.channel.Channel;
+import io.netty.util.concurrent.FastThreadLocal;
+
+import static io.servicetalk.transport.netty.internal.NettyIoExecutors.fromNettyEventLoop;
+
+final class ExecutionContextUtils {
+
+    private static final FastThreadLocal<IoExecutor> CHANNEL_IO_EXECUTOR = new FastThreadLocal<>();
+
+    private ExecutionContextUtils() {
+        // No instances
+    }
+
+    static HttpExecutionContext channelExecutionContext(final Channel channel,
```

Comment:
Since this class is part of an optimization, for future-self and to avoid perf regressions consider some javadocs to explain the rationale and use case